libs/market-info/src/components/market-info/info-market.tsx

```diff
@@ -1,51 +1,43 @@
 import { useMemo } from 'react';
-import { formatNumber, t, useYesterday } from '@vegaprotocol/react-helpers';
+import {
+  formatNumber,
+  t,
+  useDataProvider,
+  useYesterday,
+} from '@vegaprotocol/react-helpers';
 import { AsyncRenderer, Splash, Accordion } from '@vegaprotocol/ui-toolkit';
 import pick from 'lodash/pick';
 import BigNumber from 'bignumber.js';
-import { useQuery } from '@apollo/client';
 import { totalFees } from '@vegaprotocol/market-list';
+import type { Schema } from '@vegaprotocol/types';
 import {
   AccountType,
   Interval,
   MarketStateMapping,
   MarketTradingModeMapping,
 } from '@vegaprotocol/types';
-import { MARKET_INFO_QUERY } from './info-market-query';
-import type {
-  MarketInfoQuery,
-  MarketInfoQuery_market,
-  MarketInfoQuery_market_candlesConnection_edges,
-} from './__generated__/MarketInfoQuery';
 import { MarketInfoTable } from './info-key-value-table';
 import { ExternalLink } from '@vegaprotocol/ui-toolkit';
 import { generatePath } from 'react-router-dom';
 import { useEnvironment } from '@vegaprotocol/environment';
 import { Link as UiToolkitLink } from '@vegaprotocol/ui-toolkit';
 import Link from 'next/link';
+import { marketInfoDataProvider } from './market-info-data-provider';
 
 const Links = {
   PROPOSAL_PAGE: ':tokenUrl/governance/:proposalId',
 };
 
 export interface InfoProps {
-  market: MarketInfoQuery_market;
+  market: Schema.Market;
```

Comment:
chore (blocking): Why this change? We should be using actual generated query type here as if someone changes a field the component will still expect the entire schema.

Same goes for other uses of types directly from Schema. You may want to create type aliases if the types are super long and make the code hard to read

Reply:
agree, updated